tests/check/test-basic.sh

```diff
@@ -45,6 +45,9 @@ ostree admin --sysroot=sysroot deploy --karg=root=LABEL=MOO --karg=quiet --os=te
 assert_status_jq '.deployments[0].version == "1.0.10"'
 echo "ok status shows right version"
 
+rpm-ostree reload
+echo "ok reload"
+
```

Comment:
Maybe combine this with the ONE BILLION DOLLARS test case below to see that it actually works? I.e. swap out the killing for `reload` (and maybe try the upgrade before *and* after, rather than just after).